src/Middleware/Diagnostics/src/DeveloperExceptionPage/DeveloperExceptionPageMiddlewareImpl.cs

```diff
@@ -179,18 +179,11 @@ private async Task DisplayExceptionContent(ErrorContext errorContext)
                 Status = httpContext.Response.StatusCode
             };
 
-            var exceptionDetails = _exceptionDetailsProvider.GetDetails(errorContext.Exception);
             problemDetails.Extensions["exception"] = new
             {
-                Details = exceptionDetails.Select(d => new
-                {
-                    Message = d.ErrorMessage ?? d.Error?.Message,
-                    Type = TypeNameHelper.GetTypeDisplayName(d.Error),
-                    StackFrames = d.StackFrames,
-                }),
+                Details = errorContext.Exception.ToString(),
```

Comment:
There's nothing expecting details to be an array, right?

Reply:
Don't believe so, this is simply developer-time formatting